libbeat/common/jsontransform/jsonhelper.go

```diff
@@ -8,35 +8,35 @@ import (
 	"github.com/elastic/beats/libbeat/logp"
 )
 
-func WriteJSONKeys(event common.MapStr, keys map[string]interface{}, overwriteKeys bool, errorKey string) {
+func WriteJSONKeys(event common.MapStr, keys map[string]interface{}, overwriteKeys bool) {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function WriteJSONKeys should have comment or be unexported